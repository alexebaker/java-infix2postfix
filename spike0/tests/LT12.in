
a;
1;
1+2;2+3;
3+
4;
 4
 +

  5 ;6;1+2-3+4*5/6*7+8-9+0;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             ----------
0!
-2+5/3;
1+2*3+-4;
(
 (
  ((((((b)))))))/
-
(a+r)
);


a=(b+c-d*e/f+g-h*i/j);


p=---q---r;
f=4!
+(g = 4);
x=(2*5*8+-(y=3+
4))%
;


z=--Z!%*f;

 a=b=c=d=e=f=g=h=i=j=k=l=m=n=o=p=q=r=s=t=u=v=w=x=y=z=
 A=B=C=D=E=F=G=H=I=J=K=L=M=N=O=P=Q=R=S=T=U=V=W=X=Y=Z=
 0+1+2+3+4+5+6+7+8+9+
 0*1*2*3*4*5*6*7*8*9*
 0-1-2-3-4-5-6-7-8-9-
 0/1/2/3/4/5/6/7/8/9/
 a+b+c+d+e+f+g+h+i+j+k+l+m+n+o+p+q+r+s+t+u+v+w+x+y+z+
 A-(B-(C-(D-(E-(F-(G-(H-(I-(J-(K-(L-(M-(N-(O-(P-(Q-(R-(S-(T-(U-(V-(W-(X-(Y-(Z-(
 a)*b)*c)*d)*e)*f)*g)*h)*i)*j)*k)*l)*m)*n)*o)*p)*q)*r)*s)*t)*u)*v)*w)*x)*y)*z)*
 a;b;c;d;e;f;g;h;i;j;k;l;m;n;o;p;q;r;s;t;u;v;w;x;y;z;