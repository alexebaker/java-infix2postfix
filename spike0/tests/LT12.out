a
1
12+
23+
34+
45+
6
12+3-45*6/7*+8+9-0+
0!__________2-53/+
123*+4_+
bar+_/
abc+de*f/-g+hi*j/-=
pq___r__-=
f4!g4=+=
x25*8*y34+=_+%=
zZ!%__f*=
abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ01+2+3+4+5+6+7+8+9+01*2*3*4*5*6*7*8*9*0*+1-2-3-4-5-6-7-8-9-01/2/3/4/5/6/7/8/9/a/-b+c+d+e+f+g+h+i+j+k+l+m+n+o+p+q+r+s+t+u+v+w+x+y+z+A+BCDEFGHIJKLMNOPQRSTUVWXYZab*-c*-d*-e*-f*-g*-h*-i*-j*-k*-l*-m*-n*-o*-p*-q*-r*-s*-t*-u*-v*-w*-x*-y*-z*-a*-====================================================
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
